python/src/opendp/_convert.py

```diff
@@ -71,6 +71,9 @@ def py_to_c(value: Any, c_type, type_name: RuntimeTypeDescriptor = None) -> Any:
 
     if isinstance(type_name, str):
         type_name = RuntimeType.parse(type_name)
+    
+    if type_name == ExtrinsicObject:
+        type_name = "ExtrinsicObject"
```

Comment:
This is a great comment. For context, as implemented, the parsed representation of an atomic type (like "f64") is still a string ("f64"), which is why the latter code worked.

I made some adjustments to simplify the process. Mainly, ExtrinsicObjects are now passed directly over Ffi, instead of undergoing the process of being packed up into an AnyObject.